src/network/networkd-network-gperf.gperf

```diff
@@ -130,6 +130,11 @@ BridgeFDB.VLANId,                       config_parse_fdb_vlan_id,
 BridgeVLAN.PVID,                        config_parse_brvlan_pvid,                       0,                             0
 BridgeVLAN.VLAN,                        config_parse_brvlan_vlan,                       0,                             0
 BridgeVLAN.EgressUntagged,              config_parse_brvlan_untagged,                   0,                             0
+Network.PrefixDelegation,               config_parse_bool,                              0,                             offsetof(Network, router_prefix_delegation)
+PrefixDelegation.RouterLifetime,        config_parse_uint32,                            0,                             offsetof(Network, router_lifetime)
```

Comment:
Fixed the section to be IPv6PrefixDelegation, variable to RouterLifetimeSec and function config_parse_sec().